docs/runtimes/image-build.md

```diff
@@ -150,6 +150,25 @@ project.build_function(
 )
 ```
 
+#### Using self-signed registry
+If you need to build your function and push the resulting container image to an external Docker registry that uses a self-signed SSL certificate,
+you can use Kaniko with the --skip-tls-verify flag.
+This flag allows Kaniko to ignore SSL certificate verification while pulling base images or pushing the final built image to the registry over HTTPS.
```

Comment:
When using this flag, Kaniko ignores the SSL certificate verification while pulling base images and/or pushing the final built image to the registry over HTTPS.